{{cookiecutter.project_slug}}/config/settings/base.py

```diff
@@ -321,6 +325,12 @@
     ),
     "DEFAULT_PERMISSION_CLASSES": ("rest_framework.permissions.IsAuthenticated",),
 }
+
+# django-cors-headers - https://github.com/adamchainz/django-cors-headers#setup
+CORS_URLS_REGEX = {
+    r"^/api.*$",
+}
```

Comment:
I think this setting should be a string, not a set? Maybe we could also add an extra `/` after the `"api"`:

```suggestion
CORS_URLS_REGEX = r"^/api/.*$"
```

https://github.com/adamchainz/django-cors-headers#cors_urls_regex